ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/api/BatchingQueueSpec.scala

```diff
@@ -150,7 +152,7 @@ class BatchingQueueSpec
       val correlatedSubmission2 = createCorrelatedSubmission("2")
       val batches = mutable.ListBuffer.empty[Seq[CorrelatedSubmission]]
 
-      val maxWaitDuration = 50.millis
+      val maxWaitDuration = 500.millis
```

Comment:
This will cause this test to take at least half a second; how about 100?

Reply:
Tried it. I can reliably reproduce about 2 failures in 100 on my machine, and CI is slower than my machine.